domain/src/main/java/life/qbic/apps/datamanager/services/UserRegistrationService.java

```diff
@@ -46,28 +57,21 @@ public void registerUser(final String fullName, final String email, final char[]
       throw new RuntimeException("User registration failed.");
     }
     // We subscribe to the domain event UserRegistered, in order to forward the event to
-    // the EventStore and to the NotificationService for broadcasting
+    // the EventStore
     DomainEventPublisher.instance()
-        .subscribe(
-            new DomainEventSubscriber<UserRegistered>() {
-              @Override
-              public Class<UserRegistered> subscribedToEventType() {
-                return UserRegistered.class;
-              }
-
-              @Override
-              public void handleEvent(UserRegistered event) {
-                var notificationId = notificationService.newNotificationId();
-                var notification =
-                    Notification.create(
```

Comment:
why is this deleted? I am missing the subscription to the domain event

Reply:
Because this was duplicated was it not?